pkgs/development/python-modules/coredis/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, async-timeout
+, deprecated
+, typing-extensions
+, packaging
+, pympler
+, wrapt
+, versioneer
+, pytestCheckHook
+, redis
+, pytest-asyncio
+}:
+
+buildPythonPackage rec {
+  pname = "coredis";
+  version = "4.14.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "alisaifee";
+    repo = pname;
+    rev = version;
+    hash = "sha256-pHCQ5dePk2VhYNf/Ka+sovIn2OAVYHnLQhPFVjKmgb4=";
+  };
+
+  postPatch = ''
+    substituteInPlace pytest.ini --replace "-K" ""
+  '';
+
+  propagatedBuildInputs = [
+    async-timeout
+    deprecated
+    typing-extensions
+    packaging
+    pympler
+    wrapt
+    versioneer
+  ] ++ versioneer.optional-dependencies.toml;
```

Comment:
This includes `nativeBuildInputs`. https://github.com/alisaifee/coredis/blob/master/pyproject.toml#L3-L9

Reply:
I have removed those inputs and it still builds -> no need for `nativeBuildInputs` here.